.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+<!-- Thank you for submitting a Pull Request. Please:
+* Read our contribution guidelines: https://github.com/microsoft/playwright/blob/master/CONTRIBUTING.md
+* Associate an issue with the Pull Request.
```

Comment:
Let's be more specific and require 

Fixes: or Issue:

in the uncommented manner.

Reply:
updated.